operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -76,6 +76,14 @@ spec:
 #    ---
 #    strategy: ""
 
+##########
+# Determines whether to show links of Grafana and Traces or not depending on user
+# access to Istio control plane pods. Kiali won't show Grafana and Tracing integration
+# in case the user doesn't have permissions to list pods of IstioNamespace.
+# When empty, its default will be `true`.
+#    ---
+#  authorize_external_services: false
```

Comment:
Should we not show the default value of 'true' here instead of 'false'?

Reply:
@mwringe I think the default is explicitly set to "false" ... when you explicitly set this value to "", it will default to "true" (or, at least, that's how I understand these couple of lines)